examples/jsm/loaders/MMDLoader.js

```diff
@@ -1108,6 +1109,14 @@ class MaterialBuilder {
 			params.emissive = new Color().fromArray( material.ambient );
 			params.transparent = params.opacity !== 1.0;
 
+			if ( ColorManagement.enabled === true ) {
```

Comment:
I'm currently reviewing the remaining explicit usage of `ColorManagement` in the example code.

Do we need this check here? This is the only loader which conditionally performs a color space conversion based on `ColorManagement.enabled`. Can't we just always assume color management and sRGB output is enabled?

Reply:
I think the check would also be fine to remove, yes. VOXLoader does something similar:

https://github.com/mrdoob/three.js/blob/673eca6de685e3ea398d3a76aaad4b90491f5f4c/examples/jsm/loaders/VOXLoader.js#L199